sandstone/apps/filebrowser/tests/python/test_posixfs.py

```diff
@@ -218,18 +219,26 @@ def test_get_dir_contents(self):
         fp = os.path.join(self.test_dir,'testfile.txt')
         open(fp,'w').close()
 
-        contents = PosixFS.get_dir_contents(self.test_dir)
+        user = getpass.getuser()
```

Comment:
An alternative approach in the [auth decorator](https://github.com/SandstoneHPC/sandstone-ide/blob/6996f3a951ffd66d69af8cfc53f9584377b06e6d/sandstone/lib/decorators.py#L13).